contracts/VoucherKernel.sol

```diff
@@ -13,7 +13,7 @@ import "./interfaces/IVouchers.sol";
 import "./interfaces/IVoucherKernel.sol";
 import {PaymentMethod, VoucherState, VoucherStatus, isStateCommitted, isStateRedemptionSigned, isStateRefunded, isStateExpired, isStatus, determineStatus} from "./UsingHelpers.sol";
 
-//preparing for ERC-1066, ERC-1444, EIP-838
+//preparing for ERC-1066, ERC-1444
```

Comment:
Whole this line can be removed, since both ERC-1066, ERC-1444 are Stagnant and we are not following that at all.

Reply:
removed.